### ISSUE TITLE
Trying to use global-json-file parameter emits a warning

### ISSUE THREAD

**Issue by shueybubbles** [actions/setup-dotnet#287]
**Description:**
Trying to use `global-json-file` parameter emits a warning

```
Warning: Unexpected input(s) 'global-json-file', valid inputs are ['dotnet-version', 'source-url', 'owner', 'config-file', 'include-prerelease']
```

**Task version:**
v2

**Platform:**
- [x ] Ubuntu
- [ ] macOS
- [ x] Windows

**Runner type:**
- [x ] Hosted
- [ ] Self-hosted

**Repro steps:**  
https://github.com/microsoft/sqlmanagementobjects/runs/6097095668?check_suite_focus=true

Relevant yml snippet:

```yaml
jobs:
  build:
    runs-on: windows-latest
    steps:
    - uses: actions/checkout@v2
    - uses: actions/setup-dotnet@v2
      with:
        global-json-file: global.json
```

**Expected behavior:**

 No warning, use the global.json

**Actual behavior:**
A warning is emitted.

**Comment by vsafonkin** [comment]
Hi @shueybubbles, you can use `actions/setup-dotnet@main` instead `v2` tag at the moment. This feature will be available for new release version of the action and `v3` tag update.

**Comment by vsafonkin** [comment]
@shueybubbles, I'm going to close this issue, I will ping you after release of new action version. Feel free to contact us if you have any concerns, thanks!

**Comment by vsafonkin** [comment]
Hi @shueybubbles, the new version `setup-dotnet` action is released, you can use it via `v2.1.0` or `v2` tag.
